templates/base/packages/nextjs/components/Header.tsx.template.mjs

```diff
@@ -97,8 +99,8 @@ export const Header = () => {
             <Image alt="SE2 logo" className="cursor-pointer" fill src="/logo.svg" />
           </div>
           <div className="flex flex-col">
-            <span className="font-bold leading-tight">Scaffold-ETH</span>
-            <span className="text-xs">Ethereum dev stack</span>
+            <span className="font-bold leading-tight">${logoText}</span>
+            <span className="text-xs">${logoDescription}</span>
```

Comment:
I know that this is already merged, but a nitpick: 

I think logoTitle + (logoSubtitle | logoDescription | logoText) is more intuitive (logoText feels the same as logoDescription)

We can ignore if it's too nitpicky :D